packages/components/src/components/Switch/Switch.stories.tsx

```diff
@@ -0,0 +1,107 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconQuestionMarkCircle } from '@frontify/fondue-icons';
+import { type Meta, type StoryObj } from '@storybook/react';
+import { useState } from 'react';
+
+import { Label } from '../Label/Label';
+import { Tooltip } from '../Tooltip/Tooltip';
+
+import { Switch } from './Switch';
+
+type Story = StoryObj<typeof Switch>;
+const meta: Meta<typeof Switch> = {
+    component: Switch,
+    tags: ['autodocs'],
+    parameters: {
+        status: {
+            type: 'in_progress',
+        },
+    },
+    argTypes: {
+        size: {
+            control: { type: 'select', options: ['small', 'medium', 'large'] },
+        },
+    },
+    args: {
+        id: 'airplane-mode',
+        'aria-label': 'Toggle airplane mode',
+    },
+};
+export default meta;
+
+export const Checked: Story = {
+    args: {
+        defaultValue: true,
+    },
+};
+
+export const Unchecked: Story = {
+    args: {
+        defaultValue: false,
+    },
+};
+
+export const Disabled: Story = {
+    args: {
+        defaultValue: false,
+        disabled: true,
+    },
+};
+
+export const Required: Story = {
+    args: {
+        defaultValue: false,
+        required: true,
+    },
+};
+
+export const WithLabel: Story = {
+    render: (args) => (
+        <div className="tw-flex tw-gap-1.5 tw-items-center">
+            <Label id="label" htmlFor="switch">
+                Switch
+            </Label>
+            <Switch {...args} id="switch" aria-labelledby="label" />
+        </div>
+    ),
+};
+
+export const WithLabelAndTooltip: Story = {
+    render: (args) => (
+        <div className="tw-flex tw-gap-1.5 tw-items-center">
+            <Label id="label" htmlFor="switch">
+                Switch
+                <Tooltip.Root>
+                    <Tooltip.Trigger>
+                        <IconQuestionMarkCircle size="16" />
+                    </Tooltip.Trigger>
+                    <Tooltip.Content>Tooltip</Tooltip.Content>
+                </Tooltip.Root>
+            </Label>
+            <Switch {...args} id="switch" aria-labelledby="label" />
+        </div>
+    ),
+};
+
+export const ExternallyControlled: Story = {
+    render: (args) => {
+        const [checked, setChecked] = useState(false);
+
+        const onChange = (checked: boolean) => {
+            setChecked(checked);
+        };
+
+        return <Switch value={checked} onChange={onChange} {...args} />;
+    },
+};
+
+export const Size: Story = {
+    render: (args) => (
+        <div style={{ display: 'flex', gap: 16, alignItems: 'center' }}>
+            <Switch {...args} size="small" />
+            <Switch {...args} size="medium" />
+            <Switch {...args} size="large" />
+        </div>
+    ),
```

Comment:
Tip: use `decorator` so you have a clean "show code"
Currently, we have a wrapping div with style shown, which is noise to the dev:
![Screenshot 2024-07-23 at 18 21 09](https://github.com/user-attachments/assets/40d53b9e-e700-44bb-9f23-df890ed8f9ba)

```suggestion
    decorators: [
        (Story) => {
            return (
                <div className="tw-flex tw-flex-col tw-gap-4 tw-items-center">
                    <Story />
                </div>
            );
        },
    ],
    render: (args) => (
        <>
            <Switch {...args} size="small" />
            <Switch {...args} size="medium" />
            <Switch {...args} size="large" />
        </>
    ),
```